r/react-scheduler/react_scheduler_ubi8.5.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package	: react 
+# Version	: v16.14.0
+# Source repo	: https://github.com/facebook/react/ 
+# Tested on     : RHEL 8.5,UBI8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Nailusha Potnuru <pnailush@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=react
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-v16.14.0}
+PACKAGE_URL=https://github.com/facebook/react/
+yum install -y yum-utils git jq
+#Install node latest version.
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+source ~/.bashrc
+nvm install --latest-npm node
+OS_NAME=`cat /etc/os-release | grep PRETTY_NAME | cut -d '=' -f2 | tr -d '"'`
+HOME_DIR=`pwd`
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+                echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 0
+fi
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+cd packages/scheduler
+PACKAGE_VERSION=$(jq -r ".version" package.json)
+# run the test command from test.sh
+if ! npm install && npm audit fix && npm audit fix --force; then
+        echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails" 
+        exit 1
+fi
+
+# Tests are N/A.
```

Comment:
just confirming - does this imply there are no automated tests available for the package?

Reply:
Yes Priya, there are no tests available in the package.

Reply:
Downgraded the node version, build is now success.